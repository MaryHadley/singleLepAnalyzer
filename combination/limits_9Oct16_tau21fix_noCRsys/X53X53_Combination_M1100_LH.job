universe = vanilla
Executable = X53X53_Combination_M1100_LH.sh
Should_Transfer_Files = YES
WhenToTransferOutput = ON_EXIT
Notification = Error
request_memory = 3072
Output = X53X53_Combination_M1100_LH.out
Error = X53X53_Combination_M1100_LH.err
Log = X53X53_Combination_M1100_LH.log
Queue 1