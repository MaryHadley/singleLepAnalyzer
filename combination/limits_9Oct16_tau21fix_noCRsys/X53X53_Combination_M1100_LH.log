000 (1297371.000.000) 10/10 19:27:08 Job submitted from host: <192.168.63.249:41533>
...
001 (1297371.000.000) 10/10 19:27:18 Job executing on host: <192.168.63.228:44891>
...
006 (1297371.000.000) 10/10 19:27:26 Image size of job updated: 1
	3  -  MemoryUsage of job (MB)
	2272  -  ResidentSetSize of job (KB)
...
006 (1297371.000.000) 10/10 19:32:26 Image size of job updated: 283788
	159  -  MemoryUsage of job (MB)
	162508  -  ResidentSetSize of job (KB)
...
006 (1297371.000.000) 10/10 20:17:30 Image size of job updated: 284596
	159  -  MemoryUsage of job (MB)
	162508  -  ResidentSetSize of job (KB)
...
005 (1297371.000.000) 10/10 20:19:22 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:51:35, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:51:35, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1633  -  Run Bytes Sent By Job
	323  -  Run Bytes Received By Job
	1633  -  Total Bytes Sent By Job
	323  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :     1101        1   3824814
	   Memory (MB)          :      159     3072      3072
...
