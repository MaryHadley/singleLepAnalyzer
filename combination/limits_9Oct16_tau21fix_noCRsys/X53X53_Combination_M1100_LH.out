/var/lib/condor/execute/dir_23640
Total number of likelihood evaluations: 55000100
Total number of likelihood evaluations: 5500100
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1100_LH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1100_LH/quant-toys:0-sig-dce7406fae.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1100_LH/quant-data-sig-812c14614c.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1100_LH
