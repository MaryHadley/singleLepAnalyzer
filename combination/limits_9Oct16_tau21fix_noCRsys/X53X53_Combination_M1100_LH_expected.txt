# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.083518   0.041759     0.1757  0.057556    0.12279
